platform/viewer/src/loadDynamicConfig.js

```diff
@@ -1,17 +1,30 @@
-export default async () => {
-  let query = new URLSearchParams(window.location.search);
-  let configUrl = query.get('configUrl');
+export default async config => {
+  const useDynamicConfig = config.useDynamicConfig;
 
-  if (!configUrl) {
-    // Handle OIDC redirects
-    const obj = JSON.parse(sessionStorage.getItem('ohif-redirect-to'));
-    if (obj) {
-      const query = new URLSearchParams(obj.search);
-      configUrl = query.get('configUrl');
+  // Check if useDynamicConfig enabled
+  if (useDynamicConfig.enabled) {
```

Comment:
dangerouslyUseDyanamicConfig

Reply:
@Ouwen Renamed `useDynamicConfig` to `dangerouslyUseDynamicConfig`